docs(kommander): Add distributed authnz pages (#2795)
mesosphere/dcos-docs-site: pages/ksphere/kommander/latest/operations/identity-providers/index.md

By default, you login to konvoy with a credential given by `konvoy up`. You can retrieve it later by using `konvoy get ops-portal`.

> These static credentials should be used only to access **operations portal** for configuring an external identity provider. Since these are static credentials and there is currently no way of updating them they should be treated only as backup credentials and not used for normal access. Always login with own identity from external identity provider that usually provide additional security features like Multi Factor Authentication.

Reviewer: ```suggestion
Static credentials should only be used to access **operations portal** for configuring an external identity provider. Since there is  no way of updating static credentials they should be treated as backup credentials and not used for normal access. Always login with your own identity from external identity providers that provide additional security features like Multi Factor Authentication.
```
